lib/fluent/supervisor.rb

```diff
@@ -385,6 +385,9 @@ def self.default_options
         standalone_worker: false,
         signame: nil,
         winsvcreg: nil,
+        winsvc_name: 'fluentdwinsvc',
```

Comment:
Is there another similar place I could add these? I couldn't find one. I can make one in fluentd?